Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -210,6 +212,19 @@ protected theorem finsum {ι : Type*} {f : ι → ℂ → E} (h : ∀ i, CircleI
 nonrec theorem neg (hf : CircleIntegrable f c R) : CircleIntegrable (-f) c R :=
   hf.neg
 
+/--
+If `f` is circle integrable, then so are its scalar multiples.
+-/
+theorem const_smul {a : A} {f : ℂ → A} (h : CircleIntegrable f c R) :
+    CircleIntegrable (a • f) c R := by
+  apply IntervalIntegrable.const_mul h
+
+/--
+If `f` is circle integrable, then so are its scalar multiples.
+-/
+theorem const_smul_fun {a : A} {f : ℂ → A} (h : CircleIntegrable f c R) :
```

Comment:
I believe [the convention](https://leanprover-community.github.io/contribute/naming.html#unexpanded-and-expanded-forms-of-functions) says `const_fun_smul`.